config/config_source.go

```diff
@@ -149,6 +162,7 @@ type FileWatcherSource struct {
 }
 
 // NewFileWatcherSource creates a new FileWatcherSource.
+// TODO: doesn't seem to be used
```

Comment:
this is used here: https://github.com/pomerium/pomerium/blob/master/internal/cmd/pomerium/pomerium.go#L49